src/app/services/open-weather.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { OpenWeatherService } from './open-weather.service';
+
+describe('OpenWeatherService', () => {
```

Comment:
probably some tests are missing to hit our 70% rate

Reply:
also there is some missing init (http service mock to even start pass the tests)